### DISCUSSION TITLE
overrideValues on hiddenFieldHandle doesn't work

### DISCUSSION THREAD

**Discussion by mazaalai** [solspace/craft-freeform#1186] [Questions & Answers] [answered]
**Describe the bug or issue you're experiencing**
I want to override a hidden field value but it is not working in Freeform 5.1.8. It was working fine in 3 and 4.

**Steps to reproduce**

1. In your twig file put following: Change renderTag and hiddenFieldHandle

{{ form.renderTag({
   overrideValues: {
      hiddenFieldHandle: entry.id
   }
}) }} 

2. Submit form
3. See submission
4. Hidden field is empty

**Expected behavior**

Overriding the value of hidden field. For my case it was entity.title

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.8.6
- Freeform Version: 5.1.8
- Freeform Edition: Lite
- Fresh Install or Upgrade: Upgrade from 4

**Additional context**
I have also tried overrideValue on non hidden field it still doesn't work.

I am using following to override my hidden field value for now.

{% block javascript %}
      <script>
          document.addEventListener('DOMContentLoaded', function() {
              var hiddenFieldHandleField = document.getElementById('form-input-hiddenFieldHandle');
              if (hiddenFieldHandle) {
                  hiddenFieldHandleField.value = "{{ entry.title }}";
              }
          });
      </script>
{% endblock %}

**Comment by kjmartens** [comment, marked as answer]
Hi @mazaalai,

This has changed in Freeform 5. You'll need to do it this way now: 🙂 

### At Form level

``` twig
{{ freeform.form("myFormHandle", {
    fields: {
        "myFieldHandle": {
            label: "My Field Label Override",
            instructions: "My field instructions override.",
            value: "my field value override",
            attributes: {
                container: {
                    "data-my-input-container": true,
                },
                input: {
                    id: "my-field",
                    class: "big-text",
                },
            },
        },
    },
}).render() }}
```

### At Field level

``` twig
{{ field.render({
    label: "My Field Label Override",
    instructions: "My field instructions override.",
    value: "my field value override",
    attributes: {
        container: {
            class: "container",
        },
        input: {
            novalidate: true,
            class: "input-element",
        },
        label: {
            class: "label",
        },
        instructions: {
            class: "instructions",
        },
        error: {
            class: "error-block",
        },
    }
}) }}
```

- https://docs.solspace.com/craft/freeform/v5/guides/passing-dynamic-data-to-forms/
- https://docs.solspace.com/craft/freeform/v5/templates/formatting/#override-field-values

**Reply by mazaalai** [reply]
Thanks for the quick response. 

By the way, I looked up Overrides in the search bar, but it does not show up.
![image](https://github.com/solspace/craft-freeform/assets/15675633/9dbe33be-6208-40fb-a96c-09d3f8a37714)
